<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rab_file</class>
 <widget class="QDialog" name="rab_file">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>664</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>911</width>
    <height>664</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>911</width>
    <height>664</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Работа с файлами</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 243, 250);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 222, 239);
color: rgb(57, 88, 134);</string>
   </property>
   <property name="text">
    <string>2. Работа с файлами (модуль os, shutil) </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_create_filepath">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(57, 88, 134);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 222, 239);
color: rgb(57, 88, 134);</string>
   </property>
   <property name="text">
    <string>Путь:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="tB_create_filepath">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 174, 224);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 222, 239);
color: rgb(57, 88, 134);</string>
   </property>
   <property name="text">
    <string>Создать пустой текстовый файл</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_create">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 174, 224);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 222, 239);
color: rgb(57, 88, 134);</string>
   </property>
   <property name="text">
    <string>Записать в файл строку</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="tB_write_filepath">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 174, 224);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_write_filepath">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(57, 88, 134);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 222, 239);
color: rgb(57, 88, 134);</string>
   </property>
   <property name="text">
    <string>Файл:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_write">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 174, 224);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Записать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 222, 239);
color: rgb(57, 88, 134);</string>
   </property>
   <property name="text">
    <string>Строка:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_write_text">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(57, 88, 134);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 222, 239);
color: rgb(57, 88, 134);</string>
   </property>
   <property name="text">
    <string>Информация</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="tE_infa">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>411</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(57, 88, 134);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_read_filepath">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(57, 88, 134);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_read">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>440</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 174, 224);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Читать</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="tB_read_filepath">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>410</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 174, 224);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 222, 239);
color: rgb(57, 88, 134);</string>
   </property>
   <property name="text">
    <string> Прочитать файл</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 222, 239);
color: rgb(57, 88, 134);</string>
   </property>
   <property name="text">
    <string>Файл:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_delete_filepath">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>530</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(57, 88, 134);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 222, 239);
color: rgb(57, 88, 134);</string>
   </property>
   <property name="text">
    <string>Файл:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delete">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>560</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 174, 224);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tB_delete_filepath">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>530</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 174, 224);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>500</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 222, 239);
color: rgb(57, 88, 134);</string>
   </property>
   <property name="text">
    <string>Удалить файл </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>620</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 174, 224);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 222, 239);
color: rgb(57, 88, 134);</string>
   </property>
   <property name="text">
    <string>Имя файла:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_create_name">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(57, 88, 134);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
